### DISCUSSION TITLE
Add default alerts to the geth charts

### DISCUSSION THREAD

**Discussion by cakrit** [netdata/netdata#12345] [Ideas]
Starting with what is already possible and explained in https://www.netdata.cloud/blog/how-to-monitor-the-geth-node-in-under-5-minutes/

We should add some predefined alerts. @CryptoFewka told me that the most important thing a user would want to be alerted on, is whether they're "falling behind". I expect he means something similar to what we describe below, but the specifics are far beyond my expertise:

<img width="737" alt="image" src="https://user-images.githubusercontent.com/43294513/157308753-113f14ec-58b1-4630-b668-bd9d22f5ed96.png">

Not sure if these alerts are on our master branch and if they work properly, starting the discussion here so we see how can improve it further.

**Comment by ilyam8** [comment]
> `#12345`

Sorry for the off-topic, but I was waiting for this number. You got lucky!
